test/test_comments.sh

```
@@ -32,6 +32,49 @@ setup_mocks_with_pr() {
+
+  setup_mocks
+
+  local reply_spec cid rdata
+  for reply_spec in "$@"; do
+    cid="${reply_spec%%:*}"
+    rdata="${reply_spec#*:}"
+    echo "$rdata" > "$mock_dir/replies_${cid}.json"
+  done
+
+  printf '#!/usr/bin/env bash\ncase "$*" in\n' > "$mock_dir/gh"
+  printf '  *\"pulls?head=acme:feature-branch\"*)\n' >> "$mock_dir/gh"
```

Shellcheck (0.11.0)

[style] Consider using { cmd1; cmd2; } >> file instead of individual redirects.

(SC2129)